lib/cdo/slack.rb

```diff
@@ -19,6 +19,18 @@ class Slack
 
   SLACK_TOKEN = CDO.slack_token.freeze
 
+  # @param email [String] The email of the Slack user.
+  # @raise [ArgumentError] If the email does not correspond to a Slack user.
+  # @return [nil | String] The mention name for the Slack user.
+  def self.mention_name(email)
```

Comment:
For (1) (ignoring (2)), it would be useful to call this `get_mention_name`, making clear that `get` is the action. But I dropped the `get`, per your previous (good stylistic) feedback.

For (2), though I sympathize, I think the mention name is separate from the mention character. That said, I explicitly checked the response for this, as I wasn't certain which way it would be.

That said, I'm happy to rename to `user_name`, which I'll do unless I hear otherwise.